src/data-tutorials/users.tsx

```diff
@@ -962,11 +962,18 @@ const Users: User[] = [
   },
   {
     title: 'How to Assemble a Web3 Dream Team and Build a High Value dApp',
-    description: 'There are multiple factors to consider when assembling a web3 dream team.',
+    description: 'Learn how to build a dapp for seliing football tickets on the celo blockchain.',
     preview: require('./showcase/beginner/How-to-Assemble-a-Web3-Dream-Team-and-Build-a-High-Value-dApp.png'),
     website: 'blog/tutorials/how-to-assemble-a-web3-dream-team-and-build-a-high-value-dapp',
     tags: ['celo', 'dapp', 'beginner', 'celosage'],
   },
+  {
+    title: 'How to Build a Full Stack Dapp For Selling Football Tickets on Celo',
+    description: 'Learn how to build a dapp for seliing football tickets on the celo blockchain',
+    preview: require('./showcase/beginner/How-To-Build-a-Fullstack-Dapp-For-Selling-Football-Tickets.png'),
```

Comment:
`./showcase/intermediate/How-To-Build-a-Fullstack-Dapp-For-Selling-Football-Tickets.png`

Reply:
Fix the Image path here.